### ISSUE TITLE
Add Hosting Landing Page

### ISSUE THREAD

**Issue by bengreeley** [WordPress/wporg-main-2022#199]
[Figma Design](https://www.figma.com/file/MFnUkVIvdcQJiohII36HR5/Hosting?node-id=638%3A3992&t=Zg5wsR4jBnrrU38B-0)
[Current hosting URL](https://wordpress.org/hosting)

| Desktop      | Mobile |
| ----------- | ----------- |
| ![Landing - Desktop](https://user-images.githubusercontent.com/7133400/217887588-bfefd76c-b134-42d3-ba1c-5640437693df.jpg)      | ![Landing - Mobile](https://user-images.githubusercontent.com/7133400/217887596-a392e624-4fb5-4c2d-b21a-20104c680ed8.jpg)
       |

This should be done as a template and pattern, similar to https://github.com/WordPress/wporg-main-2022/pull/157

**Comment by bengreeley** [comment]
This is ready for initial designer review @WordPress/meta-design . Including screenshots below. If you want to preview the page, it can be previewed at https://wordpress.org/hosting-2/ by clicking preview in the editor. Thank you.

| Desktop      | Mobile |
| ----------- | ----------- |
| ![desktop](https://user-images.githubusercontent.com/7133400/217922261-0ec87b7a-66ff-40c1-b143-a5a4d950e2ce.png) |  ![mobile](https://user-images.githubusercontent.com/7133400/217922529-5f9a02a1-5f28-4eb9-9914-47105357eb7c.png)    |

**Comment by beafialho** [comment]
Thanks @bengreeley! When I preview at https://wordpress.org/hosting-2/ by clicking preview in the editor, I still see the dark header and footer. However, from your screenshots, it looks great to me. 

The only comment I have is there should also be a border on the header as well:

| Preview | Design |
| ----- | ----- |
| <img width="1165" alt="Captura de ecrã 2023-02-10, às 14 27 16" src="https://user-images.githubusercontent.com/65220155/218116354-d5785ba3-d8d7-45cb-8fbc-4751ca3f7f92.png">  |  <img width="1320" alt="Captura de ecrã 2023-02-10, às 14 27 50" src="https://user-images.githubusercontent.com/65220155/218116409-b90ce433-10c3-4295-bdbf-3e7633bab7d9.png">  |

**Comment by jasmussen** [comment]
A tangential question: do we need the adminbar here? I feel like we should default to not showing it unless there's a good reason to include it.

**Comment by bengreeley** [comment]
Thank you both for taking a look!

@jasmussen I can't see a reason to have the admin bar. I'll look to have that removed when the page is launched.

@beafialho Thanks for pointing out that line, I totally missed that. I'll update it and get final approvals/launch with the changes.

**Comment by ryelle** [comment]
@beafialho Should that border be on the white global header itself (for all pages that use the white header), or is that specific to the hosting page? For example, we don't have that on [the Download page](https://wordpress.org/download/).

**Comment by beafialho** [comment]
Thanks for checking that detail out @ryelle. This border should be on the white global header whenever the background is also white. Same for the footer. 

The thinking here is, whenever the header background is the same color as the cover background, there should be a border to act as a visual mark. Another example: if the header was black and the cover was black as well, there should be a White border with opacity between them. Does that make sense?

I also added the border details to the [WP.org Design Library](https://www.figma.com/file/HOJTpCFfa3tR0EccUlu0CM/WordPress.org-Design-Library?node-id=101%3A4217&t=sjfCTkKWd2TiIaZ6-4).

**Comment by ryelle** [comment]
That makes sense. Can you add an issue to https://github.com/WordPress/wporg-mu-plugins/ for that update?

**Comment by beafialho** [comment]
Sure, I'll add it.

**Comment by StevenDufresne** [comment]
Note: siteground was removed.

- [ ] Remove siteground

**Comment by ndiego** [comment]
@StevenDufresne & @renintw circling back to this issue. Is it correct to assume that any additional changes can be made directly to the page that Ben created (/hosting-2), and then once finalized, will be turned into a template/pattern to replace the existing page (/hosting)? Or is there a better approach for this?

**Comment by adamwoodnz** [comment]
> Is it correct to assume that any additional changes can be made directly to the page that Ben created (/hosting-2), and then once finalized, will be turned into a template/pattern to replace the existing page

Correct! Ben's [PR](https://github.com/WordPress/wporg-main-2022/pull/203) has the base changes required.
